src/main/g8/default.properties

```diff
@@ -1,3 +1,2 @@
-repo = My Repo
-scalameta_version = maven(org.scalameta, scalameta_2.13)
+repo = scalafix-my-rules
```

Comment:
This way, I am hoping people name their repo `scalafix-*`, just like sbt plugins are `sbt-*`. https://github.com/liancheng/scalafix-organize-imports does, but not the others.